### ISSUE TITLE
Collaboration request

### ISSUE THREAD

**Issue by DonaldTsang** [exzhawk/EhViewer#2]
The five other active forks of F-Droid are:
- https://github.com/duzhaokun123/EhViewer
- https://github.com/mnxzyw/EhViewer
- https://github.com/vukpetrovich/EhViewer
- https://github.com/producer82/EhViewer
- https://github.com/axlecho/EhViewer

Is it possible to contact them for a consolidated effort into continuing EhViewer?

**Comment by exzhawk** [comment]
The first four repo have no meaningful commits, but only strings update.
The last one is heavily modified to be a generic online manga reader<sup>[1]</sup>. It might be hard to merge.

[1] https://bgm.tv/group/topic/350722

**Comment by pepijnmm** [comment]
https://github.com/NekoInverter/EhViewer
Is also a fork that is still being updated.(not sure if you could still merge if you wanted too)

**Comment by exzhawk** [comment]
It looks like he does not have a good reputation. I'm not going to merge or collaborate with him.
https://zhuanlan.zhihu.com/p/141455754
https://www.bilibili.com/read/cv6112968/

**Comment by DonaldTsang** [comment]
that is screwed up and we need to document this in English so people can understand the situation.

**Comment by exzhawk** [comment]
tldr. He committed a script which contains 'rm -rf /data' command(delete all user file) to a popular framework repo. And that caused many victims lost their data perminately. There is a flaw in release procedures of the framework, but he used the worst method to "remind" developers and users instead of a friendly warning.

> It looks like he does not have a good reputation. I'm not going to merge or collaborate with him.
> https://zhuanlan.zhihu.com/p/141455754
> https://www.bilibili.com/read/cv6112968/

**Comment by mirh** [comment]
This seems the sounder one now. 
https://github.com/Ehviewer-Overhauled/Ehviewer

**Comment by exzhawk** [comment]
> This seems the sounder one now. https://github.com/Ehviewer-Overhauled/Ehviewer

Whatever, I'll keep this fork usable.

The mentioned fork contains commits from techneko aka NekoInverter, so I won't collaborate with it.
